docs/reference/yaml.md

```diff
@@ -0,0 +1,132 @@
+# YAML format reference
+
+## Environmental variables/configuration
+
+You can deploy non-encrypted secrets and configuration via environmental variables set either in-line or via external (environment) files.
+
+> Note: external files take priority over in-line environmental variables. This allows you to specify a default and then have overrides within an external file.
+
+Priority:
+
+* environment_file - defined in zero to many external files
+
+```yaml
+  environment_file:
+    - file1.yml
+    - file2.yml
+```
+
+If you specify a variable such as "access_key" in more than one `environment_file` file then the last file in the list will take priority.
```

Comment:
I would not use `access_key` or `secret_key` to exemplify  environmental variables, it sends the wrong message, those should be secrets. cc @alexellis

Reply:
Probably was my example? Not sure, but makes sense to update - @ericstoekl - how about something else like a non-confidential configuration item?